examples/ssg-app/e2e/playwright.config.ts

```diff
@@ -0,0 +1,3 @@
+import { configurePlaywright } from "../../common/config-e2e";
+
+export default configurePlaywright("ssg-app", { isCI: !!process.env.CI });
```

Comment:
I don't think testing multiple browsers is often needed.
I mainly kept it when refactoring to have concurrent requests - that's how I figured out the waitUntil bug.
Maybe the default should be a single browser and we only enable multiple browsers for a few apps?

```suggestion
export default configurePlaywright("ssg-app", { isCI: !!process.env.CI, multipleBrowsers: false });
```

Reply:
The suggested change should be done here.

> Maybe the default should be a single browser and we only enable multiple browsers for a few apps?

That should be discussed en done in a different PR if we decide it's a good thing